packages/api/src/controllers/asset.test.ts

```diff
@@ -316,5 +324,58 @@ describe("controllers/asset", () => {
         },
       });
     });
+    describe("chunked upload", () => {
+      jest.setTimeout(10000);
+      it("should start upload, stop it, resume it on tus test server", async () => {
+        const filename = "test.mp4";
+        const path = "/tmp";
+        const filePath = `${path}/${filename}`;
+        const res = await client.post("/asset/request-upload", {
+          name: "tus-test",
+        });
+        await createMockFile(filePath, 1024 * 1024 * 10);
+        const file = fs.createReadStream(filePath);
+        const { size } = fs.statSync(filePath);
+        expect(res.status).toBe(200);
+        let { tusEndpoint } = await res.json();
+        tusEndpoint = tusEndpoint.split("/api/")[1];
+        tusEndpoint = `${client.server.host}/api/${tusEndpoint}`;
```

Comment:
The test server should be configured with an `ingest` that matches the server that it listens on. Is this not happening? Hows the `tusEndpoint` coming from the API vs what it becomes here?

Reply:
Ok I got curious about this one and tested/fixed it myself 🙈  
WDYT?
https://github.com/livepeer/studio/pull/1112/commits/bc934cd17dd4da7d78684fe85e702a22cd04fa9e